src/main/java/core/basesyntax/AvailableFigure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+enum AvailableFigure {
```

Comment:
Enum names should be in singular form, not plural. So 'AvailableFigure' should be renamed to something like 'FigureType' to better reflect its purpose.

Reply:
AvailableFigure is not plural, it is singular